docs/howto/operator.rst

```diff
@@ -435,3 +435,88 @@ More information
 
 See `Google Cloud SQL API documentation for patch <https://cloud.google
 .com/sql/docs/mysql/admin-api/v1beta4/instances/patch>`_.
+
+
+CloudSqQueryOperator
+^^^^^^^^^^^^^^^^^^^^
+
+Performs DDL or DML SQL queries in Google Cloud SQL instance. The DQL
+(retrieving data from Google Cloud SQL) is not supported - you might run the SELECT
+queries but results of those queries are discarded.
+
+You can specify various connectivity methods to connect to running instance -
+starting from Public IP plain connection through Public IP with SSL or both TCP and
+socket connection via Cloud SQL Proxy. The proxy is downloaded and started/stopped
+dynamically as needed by the operator.
+
+For parameter definition take a look at
+:class:`~airflow.contrib.operators.gcp_sql_operator.CloudSqlQueryOperator`.
+
+Since the query operator can run arbitrary query - it cannot be guaranteed to be
+idempotent. SQL query designer should design the queries in the way that they are
```

Comment:
`SQL query designer should design the queries to be idempotent.` sounds less complicated